File: app/src/main/java/com/dew/edward/retrofit2multiexe/models/Question.java
package com.dew.edward.retrofit2multiexe.models;
/*
 * Created by Edward on 6/11/2018.
 */

import com.google.gson.annotations.SerializedName;

public class Question {
    public String title;
    public String body;

    @SerializedName("question_id")
    public String questionId;

    @Override
    public String toString() {
        return title;
    }
}

File: app/src/main/java/com/dew/edward/retrofit2multiexe/models/Question.kt
package com.dew.edward.retrofit2multiexe.models

/*
 * Created by Edward on 6/11/2018.
 */

import com.google.gson.annotations.SerializedName

class Question (var title: String,
                var body: String,

                @SerializedName("question_id")
                var questionId: String
                ){

    override fun toString(): String {
        return title
    }
}
